test/framework/components/certmanager/generator.go

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Consider calling this package `dependencies` or something similar. To capture all pre-requisites and other things in here rather than having a package for each of them?

Reply:
Yeah, that's fine with me, I might name it `generators` though